core/src/main/java/org/elasticsearch/snapshots/SnapshotsService.java

```diff
@@ -244,41 +270,53 @@ public TimeValue timeout() {
     }
 
     /**
+     * Checks if a snapshot by the same name exists in the repository, either
+     * currently in progress or persisted in the repository.
```

Comment:
Yes, good catch, outdated java doc comments from how i previously implemented it.